tools/profile_apply

```diff
@@ -284,7 +289,10 @@ def apply(tabs):
         clear_form(form_id)
 
         for i, (section, rows) in enumerate(sections):
-            section_type = section[:1] == '[' and 'TILE_ROW' or 'GRID_SECTION'
+            if section == '[chart_divider]':
+                section_type = 'CHART_DIVIDER'
+            else:
+                section_type = section[:1] == '[' and 'TILE_ROW' or 'GRID_SECTION'
```

Comment:
Yes. It took me a while to understand it also. 
The *section[:1]* means the first char from section variable.